hugo.yaml

```diff
@@ -148,10 +148,14 @@ params:
         url: 'https://groups.google.com/a/kubernetes.io/group/dev'
         icon: fa fa-envelope
         desc: Discussion and help from your fellow users
-      - name: Twitter
-        url: 'https://twitter.com/K8sContributors'
-        icon: fab fa-twitter
-        desc: Follow us on Twitter to get the latest news!
```

Comment:
Why this removal? I get that Twitter is losing market share and the geopolitics but is that strong enough grounds to make the change?

Reply:
After the comms meeting it appears my definition of deprioritize is too heavy-handed.